### PR TITLE
Adding AppVeyor badge.

### PR DESCRIPTION
**Pull request by dhermes**
This should be fine to LGTM with no CI, since the README is the only change.

### CODE REVIEW

**Comment by dhermes** [comment]
See https://github.com/dhermes/google-cloud-python/blob/add-appveyor-badge/README.rst for the finished product

File: README.rst [hunk 1]
```diff
@@ -326,6 +326,8 @@ Apache 2.0 - See `LICENSE`_ for more information.
 
 .. |build| image:: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python.svg?branch=master
    :target: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python
+.. |appveyor| image:: https://ci.appveyor.com/api/projects/status/1akrbcvnw8481fce/branch/master?svg=true
```
Comment (daspecster) [inline README.rst]: I think you might want [this](https://www.appveyor.com/docs/status-badges/#badges-for-projects-with-public-repositories-on-github-and-bitbucket) instead.
`https://ci.appveyor.com/api/projects/status/github/googlecloudplatform/google-cloud-python`




Would there be any benefit to using https://shields.io?

**Comment by daspecster** [comment]
Would there be any benefit to using https://shields.io?

File: README.rst (hunk 1 above)
Comment (dhermes) [inline README.rst]: Big :+1:

I pulled that from https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-python/settings/badges and was wondering why it was so opaque

**Comment by dhermes** [comment]
@daspecster Feel free to look into shields.io, I'm fine with using the badge provided by AppVeyor for this PR.

**Comment by daspecster** [comment]
I am too, I just didn't know if you knew if it was better/worse idea. I'll take the source from the provider and skip the middle man.

**Comment by dhermes** [comment]
LGTY to merge? (No need to wait on Travis IMO, but we can if you think we should.)

**Comment by daspecster** [comment]
I think it would be good to get that weird `1akrbcvnw8481fce` ID out of the URL if possible.

**Comment by daspecster** [comment]
`https://ci.appveyor.com/api/projects/status/github/googlecloudplatform/google-cloud-python?branch=master&svg=true` should do the trick I think.

**Comment by dhermes** [comment]
@daspecster I updated it already. Sorry I forgot to say "PTAL"

**Comment by daspecster** [comment]
Oh my bad, it didn't live update for me or I missed it.
LGTM then! Yeah I don't think we need travis for this.

**Pull request merged**